charts/app-stable-diffusion/templates/ingress.yaml

```
@@ -45,3 +45,51 @@ spec:
           {{- end }}
     {{- end }}
 {{- end }}
+
+{{- if .Values.stablestudio.enabled -}}
+{{- $ssPort := .Values.stablestudio.service.port -}}
```

Since there might be multiple resources in a single yaml, do we need to separate them?

```suggestion
{{- if .Values.stablestudio.enabled -}}
---
{{- $ssPort := .Values.stablestudio.service.port -}}
```

---

yes, we should